lib/domain/selectors.ts

```diff
@@ -197,6 +198,11 @@ export const recipeSelectors: domainInformationSelector = {
         ingredientsSelector: 'li[class="wprm-recipe-ingredient"]',
         directionsSelector: 'div[class="wprm-recipe-instruction-text"]',
     },
+    'seriouseats.com' : {
+        titleSelector: 'h1.heading__title',
+        ingredientsSelector: '.section--ingredients .structured-ingredients__list-item',
```

Comment:
This appears to break on [some](https://www.seriouseats.com/potato-wedges-recipe-5217319) recipes because for some reason they use different classes for the list occasionally. I think it would probably work to just take the ingredients section, take whatever unordered list is in it, and then the list items.